provisioner/plain-v0/main.go

```diff
@@ -36,7 +36,7 @@ import (
 	olmv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
 	"github.com/operator-framework/rukpak/internal/storage"
 	"github.com/operator-framework/rukpak/internal/util"
-	"github.com/operator-framework/rukpak/provisioner/k8s/controllers"
+	"github.com/operator-framework/rukpak/provisioner/plain-v0/controllers"
```

Comment:
Very much a nit, and probably debatable anyway, so not for this PR:

We could consider making this `github.com/operator-framework/rukpak/provisioner/plain/v0` too.

Reply:
are we planning on having a separate controller for every single version? I imagine the go module versions will be separate from the bundle type version and there will be some compatibility published.